src/linpred.jl

```diff
@@ -185,15 +185,22 @@ function SparsePredChol(X::SparseMatrixCSC{T}) where T
         similar(X))
 end
 
-cholpred(X::SparseMatrixCSC) = SparsePredChol(X)
+cholpred(X::SparseMatrixCSC, ::Bool=false) = SparsePredChol(X)
```

Comment:
Is this because we don't have any implementation of the sparse Cholesky decomposition? Also, the fact that this was missing indicates that this method wasn't tested at all, right?
```suggestion
# No implementation of pivoted Cholesky decomposition is available
cholpred(X::SparseMatrixCSC, pivot::Bool=false) = SparsePredChol(X)
```

Reply:
> the fact that this was missing indicates that this method wasn't tested at all, right?

Yes

> we don't have any implementation of the pivoted sparse Cholesky decomposition?

We would have to ask someone who knows `LinearAlgebra` better. I just changed it to make the tests pass. Currently `SparsePredChol` does not allow passing `pivot` argument, so I have not changed it.